org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/CachingCompiler.scala

```diff
@@ -42,9 +42,15 @@ class CachingCompiler private (cacheFile: File, sbtReporter: Reporter, log: Logg
       .map {
         case (a, s) => (Option(a), Option(s))
       }.getOrElse((Option(SbtUtils.readAnalysis(cacheFile)), None))
-    cacheAndReturnLastAnalysis(new IncrementalCompilerImpl().compile(comps.scalac, comps.javac, in.sources, in.classpath, in.output, in.cache,
-      in.scalacOptions, in.javacOptions, o2jo(previousAnalysis), o2jo(previousSetup), lookup, sbtReporter, in.order,
-      skip = false, in.progress, in.incOptions, extra = Array(), log))
+      
+    if (!in.scalaInstallation.isHydraInstallation)
```

Comment:
This doesn't seem to work for me. I tried setting the Project-level Scala installation, but it always returns "false" (not a Hydra installation). It seems that during serialization something happens, in the debugger the installation is an instance of `LabelledScalaInstallationReplace`